r1cs/gadgets/std/src/fields/fp12.rs

```diff
@@ -837,6 +929,10 @@ where
         Ok(Self::new(c0, c1))
     }
 
+    /* low level version of two bit lookup table, outsources the quadratic term b0 * b1 in
```

Comment:
Since that each implementation from Fp2Gadget above uses the underlying FpGadget implementation, there's no need to comment the same thing again. I would remove it here and in all the other files (except fp.rs)